### PR TITLE
Fix TS 2.0 bug

### PR DESCRIPTION
**Pull request by JKillian**

### CODE REVIEW

File: src/documentation.ts [hunk 1]
```diff
@@ -85,8 +85,10 @@ export default class Documentation {
         }
 
         // Get the props signatures
-        details.properties = Object.keys(symbol.members).sort()
-            .map((name) => this.serializeDeclaration(symbol.members[name]));
+        // symbols without a `valueDeclaration` will crash things on TS 2.0, so filter these out
+        details.properties = Object.keys(symbol.members).sort().map((name) => symbol.members[name])
+            .filter((symbol) => symbol.valueDeclaration != null)
+            .map((symbol) => this.serializeDeclaration(symbol));
```
Comment (giladgray) [inline src/documentation.ts]: does this end up omitting any fields?

these APIs are notoriously undocumented, and I've seen this exact situation with other types (I think this is why classes are currently unsupported), so i'm down for a "whatever works" solution

Comment (JKillian) [inline src/documentation.ts]: I don't think so, I think it's just index signature stuff that gets skipped now. But not entirely sure.

**Comment by JKillian** [comment]
You'll have to merge and release this, since I can't!

**Comment by giladgray** [comment]
I'm the boss here!

**Pull request merged**